django-stubs/http/request.pyi

```diff
@@ -114,7 +155,46 @@ class QueryDict(MultiValueDict[str, str]):
     @encoding.setter
     def encoding(self, value: str) -> None: ...
     def __setitem__(self, key: Union[str, bytes], value: Union[str, bytes]) -> None: ...
+    def __delitem__(self, key: Union[str, bytes]) -> None: ...
+    def setlist(self, key: Union[str, bytes], list_: Iterable[Union[str, bytes]]) -> None: ...
+    def setlistdefault(self, key: Union[str, bytes], default_list: Optional[List[str]] = ...) -> List[str]: ...
+    def appendlist(self, key: Union[str, bytes], value: Union[str, bytes]) -> None: ...
+    # Fake signature (because *args is used in source, but it fails with more that 1 argument)
+    @overload
+    def pop(self, key: Union[str, bytes], /) -> str: ...
+    @overload
+    def pop(self, key: Union[str, bytes], default: Union[str, _Z] = ..., /) -> Union[str, _Z]: ...
```

Comment:
django-stubs still supports Python 3.7, but the `/` positional-only parameter syntax is supported only in Python 3.8+.

Opened PR #945 to fix this.